public/data/SarveshLimaye.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Sarvesh Limaye",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/SarveshLimaye.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/SarveshLimaye",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SarveshLimaye05",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I think we should capitalize `i` here 😅 What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

---

Yeah sure , thank you for the suggestion 😄